capable/ffi/sys/src/lib.rs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2022 MobileCoin Inc.
+
+//! FFI bindings for methods in `libsgx_capable.so`.
+
+#![no_std]
```

Comment:
if this is only from the untrusted side does this need to be no_std?
```suggestion
```

Reply:
Thanks, this is a good catch---leaving `std` in will cause enclave builds to break if they're depending on this lib.